zcash_client_backend/src/scan.rs

```diff
@@ -0,0 +1,221 @@
+use crossbeam_channel as channel;
```

Comment:
This isn't a standard library import; shouldn't it go in a different section?

Reply:
We generally group imports from all "external" libraries together.

Reply:
I do try and group `std / core / alloc` imports separately from external crate dependencies, but it's easy for an editor doing automatic imports to not pay attention to that sometimes.